mlir-sycl/include/mlir/Dialect/SYCL/IR/SYCLOps.td

```diff
@@ -238,20 +238,29 @@ def SYCL_AtomicType : SYCL_Type<"Atomic", "atomic"> {
   let assemblyFormat = "`<` `[` $dataType `,` custom<AccessAddrSpace>($addrSpace) `]` `,` `(` $body `)` `>`";
 }
 
+def SYCL_MultiPtrType : SYCL_Type<"MultiPtr", "multi_ptr"> {
+  let parameters = (ins "::mlir::Type":$dataType,
+                        "mlir::sycl::AccessAddrSpace":$addrSpace,
+                        "mlir::sycl::DecoratedAccess":$decAccess,
+                        "llvm::SmallVector<mlir::Type, 4>":$body);
+  let assemblyFormat = "`<` `[` $dataType `,` custom<AccessAddrSpace>($addrSpace) `,` custom<DecoratedAccess>($decAccess) `]` `,` `(` $body `)` `>`";
```

Comment:
Please, add a \` \` to have a space between the comma and the access address space/decorated access.

Reply:
Done